### ISSUE TITLE
Brazilian Portuguese translation of Advanced Algorithm Scripting

### ISSUE THREAD

**Issue by raisedadead** [freeCodeCamp/TranslationExpansion#54]
- [ ] Validate US Telephone Numbers
- [ ]  Symmetric Difference
- [ ]  Exact Change
- [ ]  Inventory Update
- [ ]  No repeats please
- [ ]  Friendly Date Ranges
- [ ] Make a Person
- [ ] Map the Debris
- [ ] Pairwise

**Comment by git-manuel** [comment]
**Working** on it !!!!

**Comment by git-manuel** [comment]
I looked at '_\seed\challenges_' directory. Can't find the JSON of the 'Advanced Algorithm' to work on.

**Comment by raisedadead** [comment]
Hi @netstudenton thanks for your interest in contributing.

The challenges have moved to
https://github.com/freeCodeCamp/freeCodeCamp/blob/staging/seed/challenges/08-coding-interview-questions-and-take-home-assignments/coding-interview-algorithm-questions.json

You can search on the repo with challenge name in case you face issues. Also feel free to reach us in the Contributors chat room.

Thanks again, and Happy translating!

**Comment by git-manuel** [comment]
Hi @raisedadead,
Someone had already make translation to the challenges. I just needed to fix some typos.

The challenges _'Make a Person'_ and  _'Map the Debris'_ are in another file,  more precisely _(02-javascript-algorithms-and-data-structures\intermediate-algorithm-scripting.json)_. In this case. should  I change this file too ?  On the same PR?

 I couldn't find the challenge  _'Friendly Date Ranges'_ anywhere. I've opened all JSON files and I used the "**Find All in All documents Opened Documents**" from notepad++ and couldn't find  '_Friendly Date Ranges'_  challenge.

Thank you.
